src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtsController.java

```diff
@@ -69,4 +71,30 @@ public AdminCourtsController(final AdminService adminService) {
     public ResponseEntity<Court> updateCourtBySlug(@PathVariable String slug, @RequestBody Court updatedCourt) {
         return ok(adminService.save(slug, updatedCourt));
     }
+
+    @GetMapping(path = "/{slug}/courtPhoto")
+    @ApiOperation("Find the photo for a court")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Successful", response = String.class, responseContainer = "List"),
```

Comment:
Only a small swagger thing, change 

responseContainer = "List"), to responseContainer = "String"),

For this one and the below